lib/fob/page_break.ex

```diff
@@ -43,6 +53,22 @@ defmodule Fob.PageBreak do
     {page_break, order.maybe_expression}
   end
 
+  defp cast_type(query, column, value) do
+    with {_table_name, schema} <- query.from.source,
+         true <- function_exported?(schema, :__changeset__, 0),
+         %{^column => type} <- schema.__changeset__() do
```

Comment:
I think you could do:
``` elixir
schema.__schema__(
      :type,
      column
    )
```
https://github.com/NFIBrokerage/agora/blob/main/lib/agora/intercontextual/tracking_board/query_helpers.ex#L219

Reply:
https://github.com/elixir-ecto/ecto/pull/3769 has some discussion about this: it looks like in the past (er, the present I guess), ecto specifically did not include virtual fields in that lookup, and in this case I need it for a virtual field. It will be possible in ecto 3.8.0 to say `schema.__schema__(:virtual_type, column)` but I can't wait for that, I'm not lazy enough :P

Reply:
good to know 👍